static/app/components/events/autofix/autofixOutputStream.tsx

```diff
@@ -0,0 +1,149 @@
+import {useEffect, useRef, useState} from 'react';
+import {keyframes} from '@emotion/react';
+import styled from '@emotion/styled';
+import {AnimatePresence, motion} from 'framer-motion';
+
+import type {AutofixStep} from 'sentry/components/events/autofix/types';
+import {IconArrow} from 'sentry/icons';
+import {space} from 'sentry/styles/space';
+import testableTransition from 'sentry/utils/testableTransition';
+
+interface Props {
+  step: AutofixStep;
+}
+
+const shimmer = keyframes`
+  0% {
+    background-position: -1000px 0;
+  }
+  100% {
+    background-position: 1000px 0;
+  }
+`;
+
+export function AutofixOutputStream({step}: Props) {
+  const [displayedText, setDisplayedText] = useState('');
+  const intervalRef = useRef<number>();
+  const previousText = useRef('');
+  const currentIndexRef = useRef(0);
+
+  useEffect(() => {
+    if (!step.output_stream) {
+      return undefined;
```

Comment:
```suggestion
      return;
```
Return by itself should already return `undefined`

Reply:
See following comments, might not need this at all,